github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -207,7 +207,10 @@ periodics:
           - "/usr/local/bin/runner.sh"
           - "/bin/sh"
           - "-c"
-          - "automation/conformance.sh"
+          - >
+            git clone https://github.com/kubevirt/kubevirt.git &&
```

Comment:
You can make prow clone it for you with `extraRefs` field [1].

[1] https://github.com/kubernetes/test-infra/blob/master/prow/config/jobs.go#L432

Reply:
What about now? I'm not sure if this would make sure `automation/conformance.sh` is exectuted in the checked out directory